### ISSUE TITLE
FR: adjust character speed with encumbrance and armour effects

### ISSUE THREAD

**Issue by wmve** [Rughalt/D35E#674]
i'm tagging this as a Feature Request because this feature doesn't yet appear to have been implemented (based on my experimentation).

**requesting:** speeds (land, climb, swim, burrow, fly) displayed on character sheet be adjusted for equipped armour and encumbrance
**requesting:** each displayed speed be displayed as "30 / 120" where 30 is "speed" determined below, and 120 is ""maxRun" determined below (perhaps you wish to drop this part, but i thought it would be the next logical step)

pseudocode:

```
// given { 
//    load - determined from inventory
//    maxload - determined from Strength
//    maxSpeed - the listed speed for the basic race
//    armour collection - from character inventory
// } 
// we want to determine determine { speed, maxRun }

if ( load / maxLoad > 2 ) 
  speed = 0
  maxRun = 0
else
  if ( load / maxLoad >1 )
    speed = 5
    maxRun = 0
  else 
    if ( any equipped armour is heavy OR (3 * load / maxLoad) > 2 )
      // "heavy" speed
      speed = int((maxSpeed+2.5)/5)*5 + int(maxSpeed/100)*5
      maxRun = 3 * maxSpeed
    else
      if ( any equipped armour is medium OR (3 * load / maxLoad) > 1 )
         // "medium" speed
        speed = int((maxSpeed+2.5)/5)*5 + int(maxSpeed/100)*5
        maxRun = 4 * maxSpeed
      else
        // "light" speed
       speed = maxSpeed
       maxRun = 4 * maxSpeed
```

**Comment by wmve** [comment]
apologies if this duplicates an existing FR.  i hunted around here and on discord but i couldn't find anything.  i thought i had hear people mention it.  this isn't a high priority, but i thought i log it so it's written up.  perhaps it will align with some other features you want to implement in some future release.

**Comment by Rughalt** [comment]
Closed, will be out in 0.95
